docs/views/documentation_template.html

```diff
@@ -1,7 +1,7 @@
 {% extends "layout.html" %}
 
 {% block pageTitle %}
-Documentation - GOV.UK Prototype Kit
+{{ title or "Documentation" }} – GOV.UK Prototype Kit
```

Comment:
Sorry, @NoraGDS - I should've mentioned this.

All documentation markdown files should be using this template, and pass their "title" in.

So the "- GOV.UK Prototype Kit" is automatically added to each one. If you view the deploy preview on this PR, you can hover over the tabs in your browser and you can see the full title.

So I _think_ these should all be fine, but won't be able to check till I'm back in the new year.